### PR TITLE
[16.0][MIG] website_snippet_product_category: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT44390

@chienandalu @CarlosRoca13 please review :)

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Functional and code review :+1:

File: website_snippet_product_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+You can select what categories can be shown by the snippet and an image for the category:
+
+#. Go to website (backend) > Configuration > Products > eCommerce Categories
```
Comment (CarlosRoca13) [inline website_snippet_product_category/README.rst]:
```suggestion
#. Go to website (backend) > eCommerce > Products > eCommerce Categories
```

**Review by CarlosRoca13 — APPROVED** [review]
Just a non-blocking comment

File: website_snippet_product_category/README.rst (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply website_snippet_product_category/README.rst]: thanks :)

**Pull request merged**